### DISCUSSION TITLE
hasura plugin install pro failed

### DISCUSSION THREAD

**Discussion by haipq7641** [hasura/graphql-engine#8963] [Question] [answered]
When i run cmd `hasura plugin install pro`
i got error
```
FATA[0002] failed to install plugin "pro": plugin "pro" does not offer installation for this platform 
```
I using Macbook with Mac OS Monterey

Thanks

**Comment by meetzaveri** [comment, marked as answer]
Hey @phamquyhai 

Thanks for reporting. This conversation looks like candidate for Github Issue. So I've gone ahead and converted this discussion into GH issue. You can check it here - https://github.com/hasura/graphql-engine/issues/8966
